grumphp.yml.dist

```diff
@@ -0,0 +1,36 @@
+imports:
+  - { resource: ./vendor/openeuropa/code-review/dist/base-conventions.yml }
+
+grumphp:
+  git_hook_variables:
+    EXEC_GRUMPHP_COMMAND: 'docker-compose exec -T web'
+
+  # Tasks.
+  tasks:
+
+    # PHP Code Sniffer.
+    phpcs:
```

Comment:
I'd add PHPstan here, at least at level 5. I managed to run it quickly with `composer require --dev phpstan/phpstan`, and adding this config from GrumPHP docs:

```
   phpstan:
      autoload_file: ~
      configuration: ~
      level: 5
      force_patterns: []
      ignore_patterns: []
      triggered_by: ['php']
      memory_limit: "-1"
      use_grumphp_paths: true
```